### PR TITLE
Countrie update

### PR DESCRIPTION
**Pull request by Zen4All**
The Netherlands Antilles have been split up in 2010 in 3 different
parts, each with an official ISO code.

I left id 151 for The Netherlands Antilles intact and updated this with Bonaire, Sint Eustatius and Saba. This way existing orders will still be displayed, and not give an error.

### CODE REVIEW

**Comment by Zen4All** [comment]
For some reason, all the merge pull request # also came with this pull, but they can be ignored

File: zc_install/sql/updates/mysql_upgrade_zencart_160.sql [hunk 1]
```diff
@@ -88,6 +88,10 @@ UPDATE countries set address_format_id = 5 where countries_iso_code_3 IN ('BEL',
 UPDATE countries set countries_name = 'Åland Islands' where countries_iso_code_3 = 'ALA';
 UPDATE countries set countries_name = 'Réunion' where countries_iso_code_3 = 'REU';
 UPDATE countries set countries_name = "Côte d'Ivoire" where countries_iso_code_3 = 'CIV';
+UPDATE countries set countries_name = 'Bonaire, Sint Eustatius and Saba' 'BQ','BES','1');
```
Comment (ajeh) [inline zc_install/sql/updates/mysql_upgrade_zencart_160.sql]: You are missing a WHERE clause on the first UPDATE. Change all of them Bonaire would not be a good idea :)

**Comment by drbyte** [comment]
:+1:

**Comment by zcwilt** [comment]
:+1:

**Pull request merged**